ui/components/app/signature-request/index.scss

```diff
@@ -68,6 +68,9 @@
 
   &__info {
     @include H7;
+
+    max-width: 90%;
```

Comment:
Where does this number come from?

Reply:
currently on signedTypeData requests from sufficiently long origins the ends of the address butt up against the sides of the notification. But I think you're right we shouldn't use an arbitrary width value here. I think it makes more sense to add left and right padding! So I'll switch to that.